duckduckgo/iOS
# meta changes to document fireproofing and update App Store images

Task/Issue URL:
 * https://app.asana.com/0/inbox/392891325557408/1192531917430355/1193122580140205
 * https://app.asana.com/0/inbox/392891325557408/1188480688641075/1193180492690417
 

Tech Design URL:
CC:

**Description**:

* Extract function and document how we are preserving DDG cookies in response to #670 
* Update the screenshot images to show new iPad layout

**Steps to test this PR**:
1. Smoke test - ensure fireproof functionality still works as expected. 
1. Ensure new images match dimensions and names of previous images.


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
